grails-app/i18n/messages.properties

```diff
@@ -65,3 +65,10 @@ job.failure.defaultMessage=An unexepected problem has occurred.
 job.error.email=Please supply a valid email address
 job.emailAddress.nullable=Please enter a valid email address so we can notify you when the job is complete.
 job.emailAddress.email.error=Please supply a valid email address
+
+job.id.label=Job Id:
```

Comment:
yep.  It did bother me putting this in there as well.  But I was more concerened with what beautiful mean't at the time.  Can be a separately configurable property used by the tag perhaps.